app/models/comment.rb

```diff
@@ -2,5 +2,9 @@ class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: :author_id
   belongs_to :post
 
+  def update_comments_counter
+    post.increment!(:commentsCounter)
+  end
+
   scope :update_comment_counter, -> { Post.find(3).update(commentsCounter: Comment.where(post_id: 3).size) }
```

Comment:
- [ ] Kindly remove this scope. There is no need having duplicate methods that do the same thing.